utopia-remix/app/util/methods.server.ts

```diff
@@ -0,0 +1,6 @@
+const methods = ["GET", "POST", "PUT", "DELETE", "OPTIONS"] as const;
+export type Method = (typeof methods)[number];
+
+export function isMethod(s: string): s is Method {
```

Comment:
What is this for? It misses out on some standard ones like `PATCH` at least...

Reply:
> It misses out on some standard ones like PATCH at least 

sure, I just included the methods we actually support in the endpoints, so I omitted PATCH/HEAD and the other ones, but I can add them (`PATCH` makes sense to add, probably `HEAD` or `CONNECT` don't 😅 )

> What is this for?

I added that utility method so we can instantiate type-safely the handler methods (https://github.com/concrete-utopia/utopia/blob/799a48208aea95a6c1412f5ecb72781dae3e8321/utopia-remix/app/util/api.server.ts#L29), and then check them as well (https://github.com/concrete-utopia/utopia/blob/799a48208aea95a6c1412f5ecb72781dae3e8321/utopia-remix/app/util/api.server.ts#L36) against the same type. WDYT?

Reply:
https://github.com/concrete-utopia/utopia/pull/4807/commits/042acfb2c3c3dab3712585533c06e2d2e6c72286